companion/src/firmwares/curvereference.cpp

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (C) OpenTX
+ *
+ * Based on code named
+ *   th9x - http://code.google.com/p/th9x
+ *   er9x - http://code.google.com/p/er9x
+ *   gruvin9x - http://code.google.com/p/gruvin9x
+ *
+ * License GPLv2: http://www.gnu.org/licenses/gpl-2.0.html
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+
+#include "curvereference.h"
+
+#include "helpers.h"     // for Helpers::getAdjustmentString()
+#include "radiodata.h"  // for ModelData
+
+QString CurveReference::toString(const ModelData * model, bool verbose) const
+{
+	if (value == 0) {
```

Comment:
2 chars indent, please :)

Reply:
Gack!

I was wondering if I should just move `CurveReference` into `io_data` along with all the classes that actually use it. Though `io_data` is getting a bit crowded already.  Need to balance between too many files and too many classes in one file...